extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
```

Comment:
Can we drop `FormData` usage then? I'd rather keep the same implementation so we don't get weird bugs in when using `GET` that we don't get with `POST`.

- - -

Can you check this with the [jQuery implementation](https://github.com/jquery/jquery/blob/e4fd41f8fa4190fbbb6cb98cf7ace64f6e00685d/src/serialize.js#L96-L126)?